lms/djangoapps/grades/tasks.py

```diff
@@ -24,9 +26,27 @@
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
 def recalculate_subsection_grade(
+        # pylint: disable=unused-argument
         user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs
 ):
     """
+    Shim to allow us to modify this task's signature without blowing up production on deployment.
```

Comment:
ah.  that makes sense.  great idea.
We should still rename it though since it's not an underscore-like function.  You could add a version number instead like recalculate_subsection_grades_v2.  And if this design pattern works, we can keep incrementing the versions (and delete old versions once newer ones are deployed).